### ISSUE TITLE
Timezone Support

### ISSUE THREAD

**Issue by diegofcoelho** [chrisspen/django-chroniker#88]
I have noticed that chroniker if following UTC time for both scheduling and recalculating dates and I was wondering if you could add support to provide a standard timezone or get it from django when convenient.

My settings.py are properly configured:

```
TIME_ZONE = 'America/Fortaleza'
USE_I18N = True
USE_L10N = True
USE_TZ = True
```
But I believe chroniker itself is responsible for calculating the time shift between zones. At least is what I believe this StackOverFlow user has shown when explaining about a related problem (not to chroniker) in this [answer](http://stackoverflow.com/a/38566337/4602178).

Did I miss something during chroniker installation?

**Comment by billyquith** [comment]
Yes, this would be nice. Times may want to be in local time as UTC won't shift for daylight saving and you may want to run something after the working day ends.

**Comment by rfenzo** [comment]
Hi, is there any update on this? 

When i change my database and django timezone to any other than UTC, the scheduler set the next execution based on utc and not my chosen timezone. Example, If the difference in timezone is 2 hours and the job executes every minute, after the first execution, the next execution will be set in 2 hours and one minute.

Maybe I'm missing something?

**Comment by chrisspen** [comment]
Confirmed and fixed in #406. Sorry it took this long.

You were right that chroniker was doing its own timezone handling and getting it wrong. `utils.localtime()` only called `make_aware()`, which returns an already-aware datetime *unchanged*. Since values are stored in UTC they are already aware, so they came back out still in UTC:

```
TIME_ZONE          : America/New_York
stored (UTC)       : 2026-08-18 20:50:51+00:00
utils.localtime()  : 2026-08-18 20:50:51+00:00   <-- what the admin displayed
django localtime() : 2026-08-18 16:50:51-04:00   <-- what it should be
```

So your `TIME_ZONE = America/Fortaleza` was being ignored for display, exactly as you observed.

#406 converts properly. Only display is affected — scheduling and storage stay in UTC, so this does not change when your jobs actually run.

@billyquith — this also covers your daylight-saving point: `timezone.localtime()` applies the correct offset for the date in question rather than a fixed one.
